src/main/java/org/folio/inventory/DataImportConsumerVerticle.java

```diff
@@ -13,6 +13,8 @@
 import static org.folio.DataImportEventTypes.DI_INVENTORY_ITEM_CREATED;
 import static org.folio.DataImportEventTypes.DI_INVENTORY_ITEM_MATCHED;
 import static org.folio.DataImportEventTypes.DI_INVENTORY_ITEM_NOT_MATCHED;
+import static org.folio.DataImportEventTypes.DI_MARC_FOR_UPDATE_RECEIVED;
+import static org.folio.DataImportEventTypes.DI_MARC_FOR_UPDATE_RECEIVED;
```

Comment:
Is it duplicated?

Reply:
doublechecked, yes import has been duplicated, thanks! deleted duplicated import